policyengine_us/variables/gov/states/ct/tax/income/credits/ct_personal_credits.py

```diff
@@ -0,0 +1,32 @@
+from policyengine_us.model_api import *
+
+
+class ct_personal_credits(Variable):
```

Comment:
something named this way would be a dollar-value credit. this needs to be multiplied by something to get that

Reply:
they call it the personal credit as it is used in computing in the[ income tax](https://github.com/PolicyEngine/policyengine-us/pull/2586) PR we would not multiply it in this variable but in the ct_income_tax_after_personal_credits variable

Reply:
OK then let's rename to `ct_personal_credit_rate` and add `unit = "\1"`